### ISSUE TITLE
Zero vairance_noise

### ISSUE THREAD

**Issue by shawnye1994** [microsoft/soc-fine-tuning-sd#4]
https://github.com/microsoft/soc-fine-tuning-sd/blob/489f904a824fa3f6f2700f3b9a36826318f4d2ac/src/trainers/soc_trainer.py#L240

Hi, Thank you very much for your open sourced code!

In the soc_trainer.py, could you please explain why do we pass a fully ZERO variance_noise to the self.soc_pipeline.scheduler.step? Based on my understanding, zero variance_noise leads to a deterministic (probability-flow ode, even with eta=1) sampling for the DDIM scheduler, which should not be "Memoryless".

I will be much appreciated for your help!
